bench/db_bench.cc

```diff
@@ -258,34 +388,41 @@ class Stats {
         bytes_ += n;
     }
 
-    void Report(const Slice &name) {
+    float get_micros_per_op()
+    {
         // Pretend at least one op was done in case we are running a benchmark
         // that does not call FinishedSingleOp().
         if (done_ < 1) done_ = 1;
+    return seconds_ * 1e6 / done_;
+    }
 
+    float get_ops_per_sec()
+    {
+        // Pretend at least one op was done in case we are running a benchmark
+        // that does not call FinishedSingleOp().
+        if (done_ < 1) done_ = 1;
+        double elapsed = (finish_ - start_) * 1e-6;
+
+        return done_ / elapsed;
+    }
+
+    float get_throughput()
```

Comment:
maybe some comment in what units the value is returned? (or include it in the function name?)